src/ui/Windows/res/PasswordSafe3.rc2

```diff
@@ -262,8 +262,18 @@ BEGIN
   IDS_DEFAULTPWLENGTHTOOSMALL "Password length is less than sum of 'at least' constraints"
   IDS_CANTSPECIFYMINNUMBER "Sorry, can't specify a minimum number of characters with 'pronounceable' option. Specified values ignored."
   IDS_OWNSYMBOLSMISSING    "If you specify using your own symbol list, you cannot leave it empty. Unselecting use of symbols."
-  IDS_EXCLUDE_FROM_SCR_CAP_OVERRIDE_WARNING_BEFORE "Screen capture protection disabled by command line.\n\nContact support if this is unexpected.\n\nYou can click OK to continue in %d seconds."
-  IDS_EXCLUDE_FROM_SCR_CAP_OVERRIDE_WARNING_AFTER  "Screen capture protection disabled by command line.\n\nContact support if this is unexpected.\n\nClick OK to continue."
+
+  IDS_SCRCAP_EXCLUDE_OVERRIDE_WARNING_BEFORE_INVALID "Screen capture protection disabled by unknown cause (state error).\n\nContact support if this is unexpected.\n\nYou can click OK to continue in %d seconds."
+  IDS_SCRCAP_EXCLUDE_OVERRIDE_WARNING_BEFORE_NONE "Screen capture protection is enabled."
+  IDS_SCRCAP_EXCLUDE_OVERRIDE_WARNING_BEFORE_CMDLINE "Screen capture protection disabled by command line.\n\nContact support if this is unexpected.\n\nYou can click OK to continue in %d seconds."
+  IDS_SCRCAP_EXCLUDE_OVERRIDE_WARNING_BEFORE_ICA "Screen capture protection disabled by detection of the ICA protocol (i.e., Citrix).\n\nContact support if this is unexpected.\n\nYou can click OK to continue in %d seconds."
```

Comment:
Displaying a text in case of non-console defeats the intent of this change: If the user is running PasswordSafe remotely, we need to *silently* disable the screen capture protection. Otherwise, we could have stayed with the command-line override semantics.

Reply:
> Displaying a text in case of non-console defeats the intent of this change: If the user is running PasswordSafe remotely, we need to _silently_ disable the screen capture protection. Otherwise, we could have stayed with the command-line override semantics.

OK/ACK... do you still want to present a blinking icon w/tooltip after reaching the app's main window?

A concern with having zero indications of implied disable of the protection is an attacker who finds a way to create or use RDP or ICA status as a way of disabling exclude from screen capture. Offhand, I'm not sure what attack that would be, so perhaps a non-concern, but it has not been researched so I typically consider a cautious approach if feasible. (i.e., having some indication protections reduced.)

What overall UX do you envision around blinking icon/tooltip messages? For example, we do away with upfront message, but do you want the blinking icon w/reason to go as well? From other comments, I think you are fine if these remain, want to make sure, thanks.

Reply:
If we detect we're non-console (RDP/ICA), the indication should be minor, e.g., disabled (grey), with an explanatory tooltip. No blinking, no popups.
I think this is a reasonable (not perfect) tradeoff between security and usability for the use case of a large institute with hundreds of users working via ICA.

Reply:
> If we detect we're non-console (RDP/ICA), the indication should be minor, e.g., disabled (grey), with an explanatory tooltip. No blinking, no popups. I think this is a reasonable (not perfect) tradeoff between security and usability for the use case of a large institute with hundreds of users working via ICA.

Okay, thanks, just saw this... ACK... current commit has "no startup warning" change... but blinking icon still present, let me add non-red-blinking for implied allow case.